bin/check-process.rb

```diff
@@ -261,7 +270,10 @@ def run
     procs.reject! { |p| p[:command] =~ /#{config[:exclude_pat]}/ } if config[:exclude_pat]
     procs.reject! { |p| p[:command] !~ /#{config[:cmd_pat]}/ } if config[:cmd_pat]
     procs.select! { |p| p[:vsz].to_f > config[:vsz] } if config[:vsz]
+    # Ensure RSS is over this value
     procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
+    # Ensure RSS is under this value
```

Comment:
This still looks wrong to me:
```
# Alert if RSS is over this value
 +  option :over_rss,
```

```
+    # Ensure RSS is over this value
￼      procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
￼+    # Ensure RSS is under this value
+    procs.select! { |p| p[:rss].to_f < config[:over_rss] } if config[:over_rss]
```

I would write it like this which makes more sense to me:
(drop the short and rename option)
```
# Alert if RSS is over this value
 +  option :under_rss,
```

```
+    # Ensure RSS is over this value
￼      procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
￼+    # Ensure RSS is under this value
+    procs.select! { |p| p[:rss].to_f < config[:under_rss] } if config[:under_rss]
```